R/metadata.R

```diff
@@ -453,8 +453,9 @@ get_active_packet <- function() {
 prevent_multiple_calls <- function(packet, name, call) {
   if (!is.null(packet$orderly2[[name]])) {
     cli::cli_abort(
-      c("Only one call to 'orderly2::orderly_{name}' is allowed",
-        i = "You have already called this function earlier in your orderly.R"),
+      c("Only one call to 'orderly2::orderly_{name}' is allowed"),
+        i = paste("You have already called this function earlier",
+                  "in your orderly file"),
```

Comment:
Can we do:
```
                  "in '{actual-name.R}'"),
```
that would be nicer if we hold that here. We should save this in the orderly-specific packet data perhaps while running?